app/models/user.rb

```diff
@@ -0,0 +1,10 @@
+class User < ActiveRecord::Base
+	has_many :tweets
```

Comment:
You should be using [`has_secure_password`](http://api.rubyonrails.org/classes/ActiveModel/SecurePassword/ClassMethods.html#method-i-has_secure_password) to give user's passwords and be able to authenticate against them.